edi_notification_oca/models/edi_exchange_type.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class EDIExchangeType(models.Model):
+    _inherit = "edi.exchange.type"
+
+    send_notification_on_error = fields.Boolean(
```

Comment:
```suggestion
    notify_on_process_error = fields.Boolean(
```
this is just for `process` action